fido-device-onboard.spec

```diff
@@ -66,6 +69,7 @@ install -D -m 0755 -t %{buildroot}%{dracutlibdir}/modules.d/52fdo dracut/52fdo/m
 
 %package -n fdo-init
 Summary: dracut module for device initialization
+Requires: openssl-devel >= 3.0.1-12
```

Comment:
`openssl-devel` is a `BuildRequires`. You want `openssl-libs` for runtime dependencies.

Reply:
fixed